client.go

```diff
@@ -0,0 +1,27 @@
+package izanami_client
+
+type client struct {
+	hostname     string
+	clientID     string
+	clientSecret string
+}
+
+type featureClient struct {
+	client *client
+}
+
+// New creates a new izanami client
+func New(host, clientID, secret string) *client {
```

Comment:
is it possible to ping the izanami and raise an error if not available, or auth failed ?